src/main/java/build/buildfarm/rpms/server/BUILD

```diff
@@ -0,0 +1,19 @@
+load("@rules_pkg//:rpm.bzl", "pkg_rpm")
+
+pkg_rpm(
+    name = "buildfarm-server-rpm",
+    data = [
+       "//examples:example_configs",
+       "//src/main/java/build/buildfarm:buildfarm-server_deploy.jar",
+       "//src/main/java/build/buildfarm:configs",
+       ":server-service"
+    ],
+    release = "0",
+    spec_file = "buildfarm-server.spec",
+    version = "0.1",
```

Comment:
I'm thinking we start versioning from buildfarm's current release: 1.10.0
https://github.com/bazelbuild/bazel-buildfarm/releases/

Especially if these rpms start showing up as assets for the release.

Reply:
Done. I have updated the rpm version to 1.10.0.